pages/index.js

```diff
@@ -50,9 +50,15 @@ const HomePage = () => {
     };
 
     const handleSubmit = () => {
+        if (exam == "NEET") {
+            router.push(
+                `/college_predictor?rank=${rank}&category=${category}&roundNumber=${roundNumber}&exam=${exam}`
```

Comment:
Rather than this, add a different route for `NEET` so it's clearer

`/college_predictor?exam=NEET[or JEE]...`